lectures/transformations/README.md

```diff
@@ -0,0 +1,301 @@
+<h2 align=center>Lecture 04</h2>
+
+<h1 align=center>Transformations</h1>
+
+<h3 align=center>12 Prairial, Year CCXXXI</h3>
+
+***Song of the day***: _[**Let go**](https://www.youtube.com/watch?v=kBXAp1N5UQI) by LambC(램씨) (2024)._
+
+### Sections
+
+1. [**A little help**](#part-1-a-little-help)
```

Comment:
You can actually remove this section altogether. I haven't used this `helper.cpp` file in a few semesters.